PowerShell.Common.props

```diff
@@ -96,8 +96,8 @@
     <Copyright>(c) Microsoft Corporation.</Copyright>
 
     <TargetFramework>net5.0</TargetFramework>
-    <LangVersion>8.0</LangVersion>
-    <PublishReadyToRun>true</PublishReadyToRun>
+    <LangVersion>preview</LangVersion>
+    <PublishReadyToRun Condition=" '$(Configuration)' != 'Debug' ">true</PublishReadyToRun>
```

Comment:
Please make one change per PR in case we need to backport.

Reply:
Done.

Reply:
@TravisEz13 can you update your review